src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -522,3 +560,46 @@ export async function validate8BitsTiff(tiff: Tiff): Promise<void> {
     throw new Error(`${tiff.source.url.href} is not a 8 bits TIFF`);
   }
 }
+
+export function* iterateMapSheets(bbox: BBox, gridSize: GridSize): Generator<string> {
+  const bounds = Bounds.fromBbox(bbox);
+
+  const maxX = bounds.x + bounds.width;
+  const maxY = bounds.y + bounds.height;
+
+  const tilesPerMapSheetSide = Math.floor(MapSheet.gridSizeMax / gridSize);
+
+  const tileWidth = Math.floor(MapSheet.width / tilesPerMapSheetSide);
+  const tileHeight = Math.floor(MapSheet.height / tilesPerMapSheetSide);
+
+  const stepsX = calculateSteps(bounds.x, maxX, MapSheet.origin.x, tileWidth);
+  const stepsY = calculateSteps(-maxY, -bounds.y, -MapSheet.origin.y, tileHeight); // Inverted Y axis
+
+  for (let stepY = 0; stepY < stepsY; stepY += 1) {
+    const y = maxY - stepY * tileHeight;
+    for (let stepX = 0; stepX < stepsX; stepX += 1) {
+      const x = bounds.x + stepX * tileWidth;
+      const tileName = getTileName(x, y, gridSize);
+      const tile = MapSheet.getMapTileIndex(tileName);
+      if (tile == null) {
+        logger.error({ tileName, x, y, gridSize }, `No MapTileIndex for tile ${tileName} at xy ${x}, ${y}`);
```

Comment:
does this ever happen? I feel like it should be a error?

Reply:
It can't really happen, as we are getting the `tileName` and feeding it straight into `getMapTileIndex`. 
But typescript wants me to check...
I have an idea how to implement (new MapSheet method) but probably outside the scope of this change.
Might make it an error.

Reply:
done